osbuild/util/ostree.py

```diff
@@ -153,6 +153,26 @@ def show(repo: PathLike, checksum: str) -> str:
     return msg
 
 
+def cli(*args, _input=None, **kwargs):
+    """Thin wrapper for running the ostree CLI"""
+    args = list(args) + [f'--{k}={v}' for k, v in kwargs.items()]
+    print("ostree " + " ".join(args), file=sys.stderr)
+    subprocess.run(["ostree"] + args,
+                   encoding="utf8",
+                   stdout=sys.stderr,
+                   input=_input,
+                   check=True)
+
+
+def parse_ostree_input(inputs):
+    """Parse ostree input and return the repo path and refs specified"""
+    commits = inputs["commits"]
```

Comment:
> it would be nice to stress this out somehow for anyone potentially adding new stage using this function in the future.

i.e. by enhancing the comment/documentation here in this function?

Reply:
Yeah, sorry for not being explicit. I think that enhancing the doc text is the most reasonable fix here.

While it would be also OK to pass the key to use as the function argument, it feels like too much given that the key is consistently 'commits' in all cases.

Reply:
ok I'll try to update the text in the next upload I do (possibly after another person has done a review to avoid wasting CI cycles).

If you have suggestions on what text to add here that you think would achieve the goal I would welcome them.

Reply:
An I missing something obvious here?  If the `"commits"` key can differ between stages and the only thing we need it for is to pull out `commit = inputs["commits"]`, why not have the function simply take the `commits` object itself rather than the `inputs` object with an assumed key (or a key arg)?

In other words, the function could be
```python
def parse_input_commits(commits):
    """Parse ostree input commits and return the repo path and refs specified"""
    data = commits["data"]
    refs = data["refs"]
    assert refs, "Need at least one commit"
    return commits["path"], data["refs"]
```

Reply:
@achilleas-k you are not missing anything, I like your suggestion :wink:

Reply:
ok I think I implemented the suggestion now!